cluster.yaml

```diff
@@ -0,0 +1,11 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+containerdConfigPatches:
+  - |-
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:5000"]
```

Comment:
wow this string is unreal. never seen a `"foo".bar.baz."postfix"` in my life. crazy. is this a bug?

Reply:
https://kind.sigs.k8s.io/docs/user/local-registry/

Reply:
This file will be removed in the next push.